index.html

```diff
@@ -135,15 +135,11 @@ <h2>
         Extensions to the `Navigator` interface
       </h2>
       <pre class="idl">
+        [SecureContext]
         partial interface Navigator {
           Promise&lt;BatteryManager&gt; getBattery();
         };
       </pre>
-      <p class="warning">
-        This method is not currently restricted to a <a>secure context</a>, but
-        it should be. Track progress on that in <a href=
-        "https://github.com/w3c/battery/issues/15">issue #15</a>.
-      </p>
```

Comment:
It feels like this should be turned into a note that links to https://github.com/w3c/battery/pull/51 so the interesting history is readily available. The PR comment contains the key pointers. I'm quite sure someone will ask "but why?" when this ships.

```suggestion
      <p class="note">
        This method was exposed to a non-secure context until
        <a href="https://github.com/w3c/battery/pull/51">PR #51</a>.
      </p>
```

Reply:
Makes sense, thanks. I've also updated the PR and commit messages to point to https://w3ctag.github.io/design-principles/#secure-context to provide more information about the reasoning behind this.